compiler/noirc_evaluator/src/ssa/ssa_gen/context.rs

```diff
@@ -933,7 +936,7 @@ impl<'a> FunctionContext<'a> {
         dropped_parameters.retain(|parameter| !terminator_args.contains(parameter));
 
         for parameter in dropped_parameters {
-            self.builder.decrement_array_reference_count(parameter);
+            self.builder.update_array_reference_count(parameter, false, false);
```

Comment:
Please comment in these cases (or use labeled `bool`'s) to indicate that the `update_array_reference_count`'s are incrementing, decrementing the reference count, resp.

Reply:
I think I'm just going to restructure this to avoid the extra parameters and the exposing of `update_array_reference_count`

Reply:
Done